aiida/cmdline/commands/cmd_data/cmd_remote.py

```diff
@@ -23,15 +23,15 @@
 
 @verdi_data.group('remote')
 def remote():
-    """Managing RemoteData objects."""
+    """Manipulate RemoteData objects (reference to remorte folders)."""
 
 
 @remote.command('ls')
 @arguments.DATUM(type=types.DataParamType(sub_classes=('aiida.data:remote',)))
 @click.option('-l', '--long', 'ls_long', is_flag=True, default=False, help='Display also file metadata.')
 @click.option('-p', '--path', type=click.STRING, default='.', help='The folder to list.')
 def remote_ls(ls_long, path, datum):
-    """List directory content on remote RemoteData objects."""
+    """List content of directories in a linked RemoteData object."""
```

Comment:
why "linked"? What does that mean in this context?

Reply:
I meant to say that RemoteData is a "link" to a folder (I didn't want to say "remote folder" because it might be on localhost). If it's not clear, let me know and I remove "linked"

Reply:
Ok, I fixed by removing "linked" and explaining better what a RemoteData is